api/api-doc/storage_service.json

```diff
@@ -746,6 +746,54 @@
             }
          ]
       },
+      {
+          "path":"/storage_service/backup",
+          "operations":[
+              {
+                  "method":"POST",
+                  "summary":"Starts copying sstables on object storage",
```

Comment:
```suggestion
                  "summary":"Starts copying SSTables from a specified table's snapshot to a designated bucket in object storage",
```

Reply:
Applied, thanks. But I dropped the "snapshot" mentioning, because of [these plans](https://github.com/scylladb/scylladb/pull/19890#discussion_r1695002275)